sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/export/IntervalMetricReader.java

```diff
@@ -30,12 +31,31 @@
 public final class IntervalMetricReader {
   private static final Logger logger = Logger.getLogger(IntervalMetricReader.class.getName());
 
+  private static final AtomicReference<IntervalMetricReader> globalIntervalMetricReader =
+      new AtomicReference<>();
+
   private final Exporter exporter;
   private final ScheduledExecutorService scheduler;
 
   private volatile ScheduledFuture<?> scheduledFuture;
   private final Object lock = new Object();
 
+  /**
+   * {@linkplain IntervalMetricReader#forceFlush() Force flushes} the globally registered {@link
+   * IntervalMetricReader} if available, or does nothing otherwise.
+   */
+  public static CompletableResultCode forceFlushGlobal() {
+    IntervalMetricReader intervalMetricReader = globalIntervalMetricReader.get();
+    if (intervalMetricReader != null) {
+      return intervalMetricReader.forceFlush();
+    }
+    return CompletableResultCode.ofSuccess();
```

Comment:
Just wondering - perhaps we should log (low level) debug/info here? As it's now possible to start without registering global, a customer may do exactly that and expect forceFlushGlobal to work (even though careful reading of JavaDoc explains this ;) ).